packages/test/test-end-to-end-tests/src/test/idCompressor.spec.ts

```diff
@@ -1084,3 +1085,91 @@ describeCompat("IdCompressor Summaries", "NoCompat", (getTestObjectProvider, com
 		assert(dds2.id.length > 8, "DDS's ID in attached container should not be short");
 	});
 });
+
+const dataStoreCount = 13;
+describeCompat("IdCompressor basic", "NoCompat", (getTestObjectProvider, apis) => {
+	const {
+		dataRuntime: { DataObject, DataObjectFactory },
+		containerRuntime: { ContainerRuntimeFactoryWithDefaultDataStore },
+	} = apis;
+	class TestDataObject extends DataObject {
+		public get _root() {
+			return this.root;
+		}
+
+		public get _context() {
+			return this.context;
+		}
+	}
+
+	const configProvider = createTestConfigProvider();
+	let provider: ITestObjectProvider;
+	const defaultFactory = new DataObjectFactory("T", TestDataObject, [], []);
+
+	const runtimeOptions: IContainerRuntimeOptions = {
```

Comment:
You don't need this to enable id compressor for this repro. In detached container, short IDs are always used. You just need the new feature flag I added.

Reply:
Updated